### ISSUE TITLE
Podman Desktop 1.13.2 - Podman Extension (v1.13.0-next) Fails to Start

### ISSUE THREAD

**Issue by timia2109** [containers/podman-desktop#9576]
### Bug description

I've recently updated to Podman Desktop 1.13.2, and since the update, the Podman extension (v1.13.0-next) fails to start, rendering the application unusable. Downgrading to Podman Desktop v1.12.0 resolves the issue and everything works as expected.

A colleague of mine has experienced the same behavior.

### Operating system

Windows Version 23H2

### Installation Method

Winget (Windows)

### Version

1.13.0

### Steps to reproduce

1. Update to version 1.13.2

### Relevant log output

Error: Extension podman-desktop.podman activation timed out after 20 seconds
Stack trace:
Error: Extension podman-desktop.podman activation timed out after 20 seconds
    at Timeout._onTimeout (C:\Users\XXX\AppData\Local\Programs\podman-desktop\resources\app.asar\packages\main\dist\index.cjs:151:67168)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)

**Comment by Robo210** [comment]
I ran into this issue on one of my computers. When this issue was occurring, Podman Desktop said Podman wasn't working, but I could still use it through the CLI.

I was able to resolve the issue by either reinstalling the podman CLI or recreating the podman machine; I did both and I don't remember which ultimately fixed the extension.

**Comment by timia2109** [comment]
I can confirm that this issue is limited to Podman Desktop; the CLI is still functioning correctly.

However, resetting the Podman machine solely for a GUI bug doesn’t seem like an acceptable solution.

**Comment by tom8zds** [comment]
same issue on 1.13.3, also not able to boot podman machine with auto start option. I have to boot the machine from cli since 1.13.0

**Comment by dgolovin** [comment]
I've experienced the same issue on Windows and increasing extension activation time in settings solved it.

**Comment by mvannux** [comment]
> I've experienced the same issue on Windows and increasing extension activation time in settings solved it.

Is it possible to increase the extension activation time? how? where are the settings? thx

**Comment by ChristiaanNel** [comment]
> > I've experienced the same issue on Windows and increasing extension activation time in settings solved it.
> 
> Is it possible to increase the extension activation time? how? where are the settings? thx

Under Settings > Preferences > Extensions you can increase the Max Activation Time. I updated mine to 60 seconds and it seems to have done the trick.

**Comment by mvannux** [comment]
> > > I've experienced the same issue on Windows and increasing extension activation time in settings solved it.
> > 
> > 
> > Is it possible to increase the extension activation time? how? where are the settings? thx
> 
> Under Settings > Preferences > Extensions you can increase the Max Activation Time. I updated mine to 60 seconds and it seems to have done the trick.

Ok, I was looking for some options in the "Extension: Podman" menu item... Thanks

**Comment by mil1i** [comment]
I'm noticing a similar issue on macOS Podman Desktop. Error is slightly different. It was happening on 1.13.x, and still happening on 1.14.1. 

** EDIT
I was able to "fix" this from happening by removing my nix-flakes installation; since it seemed to be complaining about a nix application under `/private/var/run/current-system`.


```shell
A JavaScript error occurred in the main process

Uncaught Exception:
Error: Invalid package /private/var/run/current-system/Applications/draw.io.app/Contents/Resources/app.asar/Electron.app/Contents/Resources/default_app.asar
at createError (node:electron/js2c/node_init:2:2298)
at t.lstatSync [as _stat] (node:electron/js2c/node_init:2:4638)
at ReaddirpStream._formatEntry (/Applications/Podman Desktop.app/Contents/Resources/app.asar/node_modules/readdirp/index.js:151:69)
at /Applications/Podman Desktop.app/Contents/Resources/app.asar/node_modules/readdirp/index.js:89:77
at Array.map (<anonymous>)
at ReaddirpStream._read (/Applications/Podman Desktop.app/Contents/Resources/app.asar/node_modules/readdirp/index.js:89:56)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```
![Image](https://github.com/user-attachments/assets/1472b518-87ee-4d6e-962d-275c7e1d46c6)
![Image](https://github.com/user-attachments/assets/14f5fe98-ec10-4d24-a7ed-2ba9ffd9ec63)

**Comment by jochenr** [comment]
I experience the same problem with v1.14.1 on Windows.

After reinstallation of podman-desktop it works....until the next reboot. then the problem occurs again

**Comment by cheema-corellian** [comment]
I am a first time user of Podman on Windows 11 Pro and I experienced the extension activation issue as well.  I did try increasing the "Max Activation Time" to 60 seconds, and for me, that made no difference.  I did have Docker installed and running.  I wasn't sure if that caused a conflict.  Should not have.  But out of abundance of caution I uninstalled Docker and tried activating the podman extension again.  Still no go.

I was about to uninstall and reinstall Podman, but simply tried restarting the desktop app and that did the trick for me.  The extension finally activated after many tries.

I do wish there was a log file somewhere that could tell us the exact reason why the podman extension was failing to activate.  And maybe there is and I don't know about it.  I did briefly looked at the "Troubleshooting Podman on Windows" page at https://podman-desktop.io/docs/troubleshooting/troubleshooting-podman-on-windows and it did not talk about the presence of a log file.
